src/hw04_task07_evklid.py

```diff
@@ -0,0 +1,19 @@
+"""
+Даны два натуральных числа.
+Вычислите их наибольший общий делитель
+при помощи алгоритма Евклида
+(мы не знаем функции и рекурсию).
+----------------------------------
+Большее число делим на меньшее.
+Если остаток = 0, то меньшее число и есть наибольший делитель.
+Если есть остаток, то большее число заменяем на остаток.
+Повторяем пока остаток не станет =0.
+"""
+number01 = int(input('число1:'))
+number02 = int(input('число2:'))
+while number01 != 0 and number02 != 0:
+    if number01 > number02:
+        number01 = number01 % number02
+    else:
+        number02 = number02 % number01
+print(number01 + number02)
```

Comment:
it's a cheat. Algorithm doesn't expect result in a sum
you can use ternary operator instead